apps/web/src/components/Tile/Tile.tsx

```diff
@@ -0,0 +1,42 @@
+import { classed } from "@tw-classed/react";
+import { size } from "lodash";
+import React from "react";
+
+type Props = {
+  children?: React.ReactNode;
+  className?: string;
+  id?: string;
+  variant?: "red" | "green" | "gray" | "dark"; // Represents the color of the tile
+  size?: "small" | "medium" | "large" | "full" | "xsmall";
+};
+
+const backgroundClasses = {
+  red: "bg-accent-red-regular",
+  green: "bg-accent-green-light",
+  gray: "bg-accent-gray-regular",
+  dark: "bg-accent-gray-dark",
+};
+
+const sizeClasses = {
+  small: "w-64 h-64",
```

Comment:
components should never have set widths/heights otherwise you'll run into overflow issues